app/src/main/kotlin/com/codingboy/composely/ui/extensions/ComposeExtensions.kt

```diff
@@ -25,4 +27,24 @@ inline fun Modifier.applyIf(
     conditional: Boolean,
     whenTrue: Modifier.() -> Modifier,
     whenFalse: Modifier.() -> Modifier = { this }
-): Modifier = if (conditional) then(whenTrue(Modifier)) else then(whenFalse(Modifier))
\ No newline at end of file
+): Modifier = if (conditional) then(whenTrue(Modifier)) else then(whenFalse(Modifier))
+
+/**
+ * Converts a Picture object to a Bitmap.
+ *
+ * This function creates a Bitmap with the same dimensions as the Picture. It then draws the Picture onto a Canvas,
+ * which is backed by the Bitmap. The resulting Bitmap is returned.
+ *
+ * @return A Bitmap representation of the Picture.
+ */
+fun Picture.toBitmap(): Bitmap {
```

Comment:
How about supporting bitmap compress? By getting quality from 1 to 100(original size)

Reply:
Or we can provide specific size of the file.

Reply:
Thank you for your feedback. Why don't we separate this issue and address it individually?
(I think it would be good if you take the improvement. :)